### DISCUSSION TITLE
Apps that deny backup

### DISCUSSION THREAD

**Discussion by benutzer74** [seedvault-app/seedvault#185] [Q&A] [answered]
When an app denies to get backed up, does this only skip user data? Would the apk itself still get saved in the backup?

**Comment by grote** [comment, marked as answer]
When an app opts out of backup, we at least back up its APK, so it will get re-installed during restore.

**Reply by ghost** [reply]
> When an app opts out of backup, we at least back up its APK, so it will get re-installed during restore.

How about the "not recently used" apps?

https://github.com/seedvault-app/seedvault/wiki/FAQ#why-do-apps-not-get-backed-up-when-not-recently-used

**Reply by grote** [reply]
All user-installed apps will have their APK backed up, except they are blacklisted from backup by the user.
